install_config/master_node_configuration.adoc

```diff
@@ -354,9 +354,9 @@ If you are using Docker 1.9+, you may want to consider enabling parallel image p
 ----
 kubeletArguments:
   serialize-image-pulls:
-  - true <1>
+  - "false" <1>
 ----
-<1> Change to false to enable parallel pulls.
+<1> Change to true to disable parallel pulls. (This is the default config)
```

Comment:
@ahardin-rh hello, I reached here looking for the same clarification. 

Saying that 'this' is the default configuration is a bit confusing, since we've noted both cases of it being false and true in the example and comment respectively. 

Would it be better worded if we say:

By default, this flag is set to `foo` (true or false, not sure which one)